src/CribblyBackend/Controllers/DivisionController.cs

```diff
@@ -40,5 +66,31 @@ public async Task<IActionResult> Create([FromBody] Division division)
             }
             return Ok(division);
         }
+
+        /// <summary>
+        /// Adds a Team to the specified division.
+        /// </summary>
+        /// <param name="Division">The Id of the division that the Team will be added to</param>
+        /// <param name="Team">The Team that will be added to the Division</param>
+        /// <returns></returns>
+        [HttpPut("{id}/add")]
```

Comment:
I wonder if it would be better to put this at `{id}/team` and make it a PATCH? I think PUT implies that the _entire_ object is being replaced, while PATCH implies an update of part of an object.... but I also don't really know so 🤷‍♂️

Reply:
fixed